### DISCUSSION TITLE
Add docs section about upgrading

### DISCUSSION THREAD

**Discussion by nidico** [jfmengels/elm-review#152] [Ideas]
I just spend quite some stripping a false-positive issue down to a minimal reproducable testcase, until I noticed that I still had elm-review 2.11.1 installed and the bug has been fixed in 2.12.x.

I now understand - the `review` elm dependencies are managed just like any other elm project, and this also includes upgrades (e.g. through `elm-json upgrade --unsafe`). I just didn't think about it yet, as I already regularly checked whether the elm-review npm package is up-to-date (through `npm outdated`) - but well, this is another thing.

I'd propose to add a short section about how to keep "elm-review" (in its entirety) up-to-date, i.e. how to check for installed and available versions for both the elm-review CLI tool (the npm package) and the elm dependencies in `review/elm.json`, and how to perform upgrades for both.

**Comment by sparksp** [comment]
Where would you go to look for this information? Would it be the [elm packages website](https://package.elm-lang.org/packages/jfmengels/elm-review/latest) or the [npm package](https://www.npmjs.com/package/elm-review)? Or would you come to one of the GitHub repos?

**Comment by nidico** [comment]
I was thinking about the [package website README](https://package.elm-lang.org/packages/jfmengels/elm-review/latest/) , which is the [Github repo README of the main package](https://github.com/jfmengels/elm-review).
